api/pkg/apis/v1alpha1/managers/stage/stage-manager_test.go

```diff
@@ -164,19 +164,19 @@ func TestCampaignWithSingleMegativeCounterStageLoop(t *testing.T) {
 		Activation:           "test-activation",
 		Stage:                "test",
 		ActivationGeneration: "1",
-		Inputs: map[string]interface{}{
```

Comment:
Do you plan to add integration test for the trigger field validation?

Reply:
Thank you for the reminder.